foundations/html_css/flexbox/flexbox-axes.md

```diff
@@ -5,7 +5,7 @@ This section contains a general overview of topics that you will learn in this l
 - You'll learn about the 2 "axes" of a flex container.
 - You'll learn how to change those axes to arrange your content in columns instead of rows.
 
-The most confusing thing about flexbox is that it can work either horizontally or vertically, and the way some rules work changes a bit depending on which direction you are working with.
+The most confusing thing about flexbox is that it can work either horizontally or vertically and, due to the way some rules work, can change a bit depending on which direction you are working with.
```

Comment:
The sentence still reads weird to me. Can this be broken into two sentences, and perhaps be a little clearer on what exactly "can change a bit"? If you need a suggestion let me know.